anomalib/core/model/anomaly_module.py

```diff
@@ -47,6 +52,7 @@ def __init__(self, params: Union[DictConfig, ListConfig]):
         self.pixel_threshold = AdaptiveThreshold(self.hparams.model.threshold.pixel_default)
 
         self.training_distribution = AnomalyScoreDistribution()
+        self.min_max = MinMax()
```

Comment:
Do we need these two at the same time?

Reply:
Not really, it depends on the normalization method that's being used. But I figured it doesn't hurt to have it either. Same as we also always have the `pixel_threshold` attribute, even when the task type is classification.